### PR TITLE
Updated wintertodt aoe warning to use game objects…

### PR DESCRIPTION
**Pull request by ocekyled**
…instead of projectiles.

Before:
https://i.imgur.com/XuYEyHt.png

After:
Normal snowfall
https://i.imgur.com/1bKObJs.gifv

Brazier breaking
https://i.imgur.com/PVtmXAl.gifv

Brazier extinguish
https://i.imgur.com/JuUSGgD.gifv

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/aoewarnings/AoeWarningPlugin.java [hunk 1]
```diff
@@ -178,6 +183,17 @@ public void onGameObjectSpawned(GameObjectSpawned event)
 				//todo
 				CrystalSpike.add(bombLocation);
 				break;
+			case 26690: //Wintertodt SnowFall
```
Comment (Ganom) [inline runelite-client/src/main/java/net/runelite/client/plugins/aoewarnings/AoeWarningPlugin.java]: This should be added to object api, and called from that.

File: runelite-client/src/main/java/net/runelite/client/plugins/aoewarnings/AoeWarningPlugin.java [hunk 2]
```diff
@@ -199,6 +215,12 @@ public void onGameObjectDespawned(GameObjectDespawned event)
 				//todo
 				CrystalSpike.remove(bombLocation);
 				break;
+			case 26690: //Wintertodt SnowFall
```
Comment (Ganom) [inline runelite-client/src/main/java/net/runelite/client/plugins/aoewarnings/AoeWarningPlugin.java]: This should be added to object api, and called from that.

**Comment by f0rmatme** [comment]
`First-time contributor`
nice work man

**Comment by zeruth** [comment]
Nice contribution!

File: runelite-client/src/main/java/net/runelite/client/plugins/aoewarnings/AoeWarningPlugin.java (hunk 2 above)
Comment (ocekyled) [reply runelite-client/src/main/java/net/runelite/client/plugins/aoewarnings/AoeWarningPlugin.java]: Sorry, bit of a noob, where would I do that?

Comment (zeruth) [reply runelite-client/src/main/java/net/runelite/client/plugins/aoewarnings/AoeWarningPlugin.java]: runelite-api...ObjectID

Comment (ocekyled) [reply runelite-client/src/main/java/net/runelite/client/plugins/aoewarnings/AoeWarningPlugin.java]: So ignore the part where it tells me not to manually edit the file?
`/* This file is automatically generated. Do not edit. */`

Comment (zeruth) [reply runelite-client/src/main/java/net/runelite/client/plugins/aoewarnings/AoeWarningPlugin.java]: yes, find your object by id and import that

Comment (zeruth) [reply runelite-client/src/main/java/net/runelite/client/plugins/aoewarnings/AoeWarningPlugin.java]: it's actually under NullObjectID, as NULL_26690 🥂

Comment (Ganom) [reply runelite-client/src/main/java/net/runelite/client/plugins/aoewarnings/AoeWarningPlugin.java]: Come to think of it, it is quite odd that is missing from objectID

Comment (Ganom) [reply runelite-client/src/main/java/net/runelite/client/plugins/aoewarnings/AoeWarningPlugin.java]: But yeah, if its actually missing, just add it.

Comment (zeruth) [reply runelite-client/src/main/java/net/runelite/client/plugins/aoewarnings/AoeWarningPlugin.java]: No, don't do that, import it from NullObjectID

**Pull request merged**